codegen/src/test/java/org/web3j/codegen/SolidityFunctionWrapperTest.java

```diff
@@ -18,39 +18,19 @@
 import java.util.Collections;
 import java.util.List;
 
-import com.squareup.javapoet.ClassName;
-import com.squareup.javapoet.MethodSpec;
-import com.squareup.javapoet.ParameterizedTypeName;
-import com.squareup.javapoet.TypeName;
-import com.squareup.javapoet.TypeSpec;
+import com.squareup.javapoet.*;
```

Comment:
Have you applied Spotless? Import wildcards aren't allowed so this will likely fail.